lib/activity.rb

```diff
@@ -0,0 +1,34 @@
+class Activity
+
+  attr_reader :name,
+              :participants,
+              :total_cost,
```

Comment:
You should not have attr_readers for total_cost or owed - these are methods, and attr_readers are used for instance variables.